src/VisualStudio/Core/Def/LanguageService/AbstractPackage`2.cs

```diff
@@ -58,32 +58,32 @@ private async Task PackageInitializationMainThreadAsync(IProgress<ServiceProgres
             RegisterEditorFactory(editorFactory);
         }
 
-        RegisterLanguageService(typeof(TLanguageService), async cancellationToken =>
-        {
-            // Ensure we're on the BG when creating the language service.
-            await TaskScheduler.Default;
-
-            // Create the language service, tell it to set itself up, then store it in a field
-            // so we can notify it that it's time to clean up.
-            _languageService = CreateLanguageService();
-            await _languageService.SetupAsync(cancellationToken).ConfigureAwait(false);
-
-            return _languageService.ComAggregate!;
-        });
-
         var miscellaneousFilesWorkspace = this.ComponentModel.GetService<MiscellaneousFilesWorkspace>();
```

Comment:
Move this to the background too.

Reply:
I originally tried that and it threw. Let me see if I can find out where that dependency is.